modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/bigquery/Field.scala

```diff
@@ -42,27 +42,27 @@ object Field {
       case Some(types) if types.possiblyWithNull(CommonProperties.Type.Object) =>
         val subfields = topSchema.properties.map(_.value).getOrElse(Map.empty)
         if (subfields.isEmpty) {
-          Suggestion.finalSuggestion(topSchema, required)(name)
+          Suggestion.finalSuggestion(topSchema, required || types.nullable)(name)
```

Comment:
As discussed in #154 I'm fairly sure this should be `required && !types.nullable`.  To settle this disagreement I think we need a unit test that only passes with one version of the code.